src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,107 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StringTest {
+
+    @DisplayName("String의 split 메서드는")
+    @Nested
+    class Describe_split {
+        final String TEST_SEPARATOR = ",";
+
+        @DisplayName("구분자를 포함하고 있는 경우")
+        @Nested
+        class Context_with_separator_contained_string {
+
+            @DisplayName("구분자를 기준으로 나누어진 문자열 배열을 반환한다")
+            @Test
+            void it_returns_array_separated_by_separator() {
+                final String input = "1,2";
+
+                final String[] splitString = input.split(TEST_SEPARATOR);
+
+                assertThat(splitString).containsExactly("1", "2");
```

Comment:
> given/when/then 구조의 when에 실제 테스트에 필요로 하는 input은 항상 변수로 선언해두는 편이었는데, 변수 선언을 굳이 하지 않고 테스트 대상에 바로 값을 대입해도 무방할지가 궁금합니다!

넵 지역 변수 선언이 불필요하다고 생각되시면 다음과 같이 제거해도 무방하다고 생각합니다.
```suggestion
                assertThat("1,2".split(TEST_SEPARATOR)).containsExactly("1", "2");
```

Reply:
답변 감사합니다!